modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control components (Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager), edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the the minimum TLS version and the TLS ciphers required by the Kubernetes control plane components to communicate with each other.
```

Comment:
double "the" in "the the minimum"

Reply:
The TLS security profile actually defines the TLS ciphers and TLS versions to use when communicating *with* those components, not for usage between each other.

Reply:
Thank you!!